exporter/opentelemetry-exporter-prometheus-remote-write/src/opentelemetry/exporter/prometheus_remote_write/__init__.py

```diff
@@ -253,12 +253,14 @@ def _parse_metric(
         return self._convert_to_timeseries(sample_sets, resource_labels)
 
     def _convert_to_timeseries(
-        self, sample_sets: Sequence[tuple], resource_labels: Sequence
+        self, sample_sets: Mapping[tuple, Sequence], resource_labels: Sequence
```

Comment:
Just curious: how come this signature change is needed, compared to https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2784 ? Was `Sequence[tuple]` the wrong type before?

Reply:
Yes, it's initialized as defaultdict(list) in the only caller:
https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2940/files/68062d8256a28909cfc458aaaa77748f7f19f525#diff-4be6a764cd546e2cea72c7e3f029a9170e541fd7480b5df96f111e6862576879R240

Reply:
Nice, thanks!